### DISCUSSION TITLE
OPDS doesnt show all books in Title view

### DISCUSSION THREAD

**Discussion by SilhouetteCipher** [koreader/koreader#11967] [Q&A]
Is there a limit to how many pages care shown in OPDS view? When i browse my library via title i only get a max of three pages of results.

Its not a huge issue as i can browse by author which is alphabetized and then find the book i would like but im just useless at remembering authors names vs book names. Id expected the same structure as author when selecting title (select first letter then get list)

Just wanted to check I was seeing expected behavior as im on the nightly for kobo clara bw and was not sure if i was seeing a bug or not.

**Comment by Frenzie** [comment]
That sounds like a bug but I suppose the question is whose.

**Comment by SilhouetteCipher** [comment]
ill pop it in the issues feed and then at least its known, rather than being in QA here. Thanks
